java/bulk-export/bulk-export-core/src/test/java/sleeper/bulkexport/model/BulkExportLeafPartitionQuerySerDeTest.java

```diff
@@ -0,0 +1,356 @@
+                "    \"subExportId\":\"" + subExportId + "\",\n" +
+                "    \"regions\":[\n" +
+                "        {\n" +
+                "            \"key\":{\n" +
+                "                \"min\":1,\n" +
+                "                \"minInclusive\":true,\n" +
+                "                \"max\":10,\n" +
+                "                \"maxInclusive\":true\n" +
+                "            },\n" +
+                "            \"stringsBase64Encoded\":true\n" +
+                "        }\n" +
+                "    ],\n" +
+                "    \"leafPartitionId\":\"" + leafPartitionId + "\",\n" +
+                "    \"files\":[\"/test/file.parquet\"]\n" +
+                "}";
+
+        // When & Then
+        assertThatThrownBy(() -> querySerDe.fromJson(queryJson))
+                .isInstanceOf(
+                        BulkExportQueryValidationException.class)
+                .hasMessage("Query validation failed for export \"" + exportId
+                        + "\": partitionRegion field must be provided");
+    }
+
+    @Test
+    public void shouldThrowExceptionNoFiles() {
+        // Given
+        BulkExportLeafPartitionQuerySerDe querySerDe = generateQuerySerDe("my-table", schema, true);
+        String queryJson = "{\n" +
+                "    \"tableId\":\"" + tableId + "\",\n" +
+                "    \"exportId\":\"" + exportId + "\",\n" +
+                "    \"subExportId\":\"" + subExportId + "\",\n" +
+                "    \"regions\":[\n" +
+                "        {\n" +
+                "            \"key\":{\n" +
+                "                \"min\":1,\n" +
+                "                \"minInclusive\":true,\n" +
+                "                \"max\":10,\n" +
+                "                \"maxInclusive\":true\n" +
+                "            },\n" +
+                "            \"stringsBase64Encoded\":true\n" +
+                "        }\n" +
+                "    ],\n" +
+                "    \"leafPartitionId\":\"" + leafPartitionId + "\",\n" +
+                "    \"partitionRegion\":{\n" +
+                "        \"key\":{\n" +
+                "            \"min\":0,\n" +
+                "            \"minInclusive\":true,\n" +
+                "            \"max\":1000,\n" +
+                "            \"maxInclusive\":false\n" +
+                "        },\n" +
+                "        \"stringsBase64Encoded\":true\n" +
+                "    }\n" +
+                "}";
+        // When & Then
+        assertThatThrownBy(() -> querySerDe.fromJson(queryJson))
+                .isInstanceOf(
+                        BulkExportQueryValidationException.class)
+                .hasMessage("Query validation failed for export \"" + exportId + "\": files field must be provided");
+    }
+
+    private BulkExportLeafPartitionQuerySerDe generateQuerySerDe(String tableName, Schema schema,
```

Comment:
The table name parameter is never used, which seems misleading. Please could we remove it?

Reply:
Removed